content/docs/user-guide/programming/components/entity-activation-system.md

```diff
@@ -0,0 +1,90 @@
+---
+linkTitle: Entity Activation System
+title: Understanding Entity Activation and the Activation Type System
+description: Learn how to set Entity Active States, and define new Custom Activation Types.
+weight: 900
+---
+
+An *entity* in **Open 3D Engine (O3DE)** is the most primordial element in the entire system. As O3DE is an [*entity-component system (ECS)* engine](overview), the *entity* is the core of how everything works.
+
+An Entity has many purposes, but one is that it is the authority of whether it is active or not. As it processes changing its active state, it sets its components accordingly, reports its change of state, and handles bringing itself to a functional place in its new state. Outside systems rely on its state handling, and only instruct it to change, allowing it to handle it all itself.
```

Comment:
IMO we should be specific here, "it has many purposes" is kinda vague.  I'd prefer specifying exactly what it does

An Entity is made up of a list of Components and a state, which can be (Constructed, Initialized, Active, Inactive).  When the Entity's state changes state, the Entity calls the appropriate functions on the components in its list (Initialized(), Activated(), Deactivated() respectively) to let them know about the state transition.